### ISSUE TITLE
Media type selection

### ISSUE THREAD

**Issue by cirin0** [prof79/fansly-downloader-ng#29]
Can you add a selection to download media type?

For example: Photo/Video/Audio.

**Comment by prof79** [comment]
Good idea but can't promise a timeframe. Has some implications to properly introduce a new option and then skip particular downloads while properly updating the skipped-items counter.
